transformer_lens/loading_from_pretrained.py

```diff
@@ -1779,7 +1790,10 @@ def get_pretrained_state_dict(
         official_model_name = str(Path(official_model_name).resolve())
         logging.info(f"Loading model from local path {official_model_name}")
     else:
-        official_model_name = get_official_model_name(official_model_name)
+        if force_unsupported_model==False:
+            official_model_name = get_official_model_name(official_model_name)
+        else:
+            official_model_name = official_model_name
```

Comment:
This change isn't needed, since you are just setting a variable to its existing value. The previous `else` should just be changed `elif` with your check in place